environments/conda-linux-64.lock.yml

```diff
@@ -225,22 +226,21 @@ dependencies:
   - json-c=0.17=h7ab15ed_0
   - json5=0.9.25=pyhd8ed1ab_0
   - jsonpointer=2.4=py312h7900ff3_3
-  - jsonschema=4.21.1=pyhd8ed1ab_0
-  - jsonschema-specifications=2023.12.1=pyhd8ed1ab_0
-  - jsonschema-with-format-nongpl=4.21.1=pyhd8ed1ab_0
+  - jsonschema=4.17.3=pyhd8ed1ab_0
+  - jsonschema-with-format-nongpl=4.17.3=pyhd8ed1ab_0
```

Comment:
Note that `frictionless` is currently pinning to `jsonschema<4.18` and `jupyter_events` also depends on that package, which results in a downgrade of some jupyter packages below. 

See https://github.com/frictionlessdata/frictionless-py/issues/1569